src/agent.spec.ts

```diff
@@ -0,0 +1,374 @@
+import { Finding, ethers, FindingSeverity, FindingType } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress } from "forta-agent-tools";
+import agent from "./agent";
+import { NETHERMIND_DEPLOYER_ADDRESS, iface } from "./constants";
```

Comment:
Additionally, a common practice across the team is to not reuse variables for arguments from production. That's to prevent cases where a variable may be declared incorrectly in production, if its used in testing, the bot won't catch that it's broken because it is using the same value. So would suggest to declare separate constants and variables here and pass those to the bot and use in testing.